experiment/generate_tests_test.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Do the CI images have python 3?

Reply:
I did not see this file (`generate_tests_test.py`) referenced directly anywhere in the repo, or as pare of a test rule in any BUILD file. Do you know what image(s) would need to be checked?

Reply:
this is being referenced indirectly by the autobump job(s) at least, I'm not sure if there's anything else, I'm not sure what image they use.

Reply:
https://github.com/kubernetes/test-infra/blob/7390662904a6df2d576e37b1ed50e0297bbd2950/config/jobs/kubernetes/test-infra/test-infra-trusted.yaml#L755

https://github.com/kubernetes/test-infra/blob/7390662904a6df2d576e37b1ed50e0297bbd2950/config/jobs/kubernetes/test-infra/test-infra-trusted.yaml#L764

https://github.com/kubernetes/test-infra/blob/master/images/bazelbuild/variants.yaml

https://github.com/kubernetes/test-infra/commit/4b188e1c1a71ae3fcdac52b9eb25c321105ccd10

looks like yes, python 2 and 3 are installed

Reply:
👍 thanks for checking.